addons/website_event_sale/views/website_sale_templates.xml

```diff
@@ -74,8 +73,54 @@
                             </a>
                         </t>
                         <div class="m-3">
-                            <t t-if="attendees" t-call="website_event.registration_ticket_access"/>
-                            <div class="row my-3">
+                            <t t-set="event_attendees" t-value="attendee_ids_per_event.get(event, [])"/>
+                            <t t-if="event_attendees and not event.is_multi_slots">
+                                <t t-set="attendees" t-value="event_attendees"/>
+                                <t t-call="website_event.registration_ticket_access"/>
+                            </t>
+                            <div t-elif="event_attendees" t-foreach="event_attendees" t-as="slot">
+                                <div t-attf-class="d-flex flex-column flex-md-row justify-content-between mt-2">
+                                    <t t-set="attendees" t-value="event_attendees[slot]"/>
+                                    <div class="mb-3">
+                                        <div class="row">
+                                            <div class="col-auto">
+                                                <b>Start</b>
+                                            </div>
+                                            <div class="col-auto ps-0">
+                                                <span t-out="slot.start_datetime"
+                                                    t-options='{"widget": "datetime", "show_seconds": False, "tz_name": event.date_tz, "format": "medium"}'/>
+                                            </div>
+                                        </div>
+                                        <div class="row">
+                                            <div class="col-auto">
+                                                <b>End</b>
+                                            </div>
+                                            <div class="col-auto ps-0">
+                                                <span t-out="slot.end_datetime"
+                                                    t-options='{"widget": "datetime", "show_seconds": False, "tz_name": event.date_tz, "format": "medium"}'/>
+                                            </div>
+                                        </div>
+                                        <div class="row">
+                                            <span t-if="website_visitor_timezone != event.date_tz" class="small">(<t t-out="event.date_tz"/>)</span>
+                                        </div>
+                                    </div>
+                                    <div t-if="attendees" t-call="website_event.registration_ticket_access"/>
+                                </div>
+                                <small>Add it to your calendar</small>
```

Comment:
Would maybe be ideal to have a small template that takes "outlook_url" and "google_url" to avoid duplicating templating.